rslib/src/backend/search/browser_table.rs

```diff
@@ -29,65 +16,6 @@ impl browser_table::Column {
             alignment: self.alignment() as i32,
```

Comment:
For consistency, I think we should use cards_label, and maybe even be a bit more verbose, eg cards_mode_label, notes_mode_label